bitnami/discourse/values.yaml

```diff
@@ -435,15 +455,35 @@ tolerations: []
 ##
 affinity: {}
 
-##
 ## PostgreSQL chart configuration
 ##
 ## https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml
 ##
 postgresql:
-  ## Whether to deploy a postgresql server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+  ## Whether to deploy a postgresql server to satisfy the applications database requirements.
+  ## To use an external database set this to true.
   ##
-  enabled: true
+  useExistingDatabase: false
```

Comment:
This is a great idea, but I think we should be conservative in favour of standardization across other Charts available. The common approach is to use `externalX` and `X.enabled` so as to make it easy for users to use different Charts in a similar manner.

E.g, [Airflow](https://github.com/bitnami/charts/blob/ccdebcbc99403900d0248fa22adc17c450c5261a/bitnami/airflow/values.yaml#L548) or [Harbor](https://github.com/bitnami/charts/blob/ccdebcbc99403900d0248fa22adc17c450c5261a/bitnami/harbor/values.yaml#L2237) charts.

Reply:
@joancafom I think the current approach duplicates lots of values and makes the chart management much (much!) more complex. Indeed the database section and the external section share exactly the same values, but the hostname of the database if this is not automatically created. Any chance we can maybe work on the wording, but keep the current logic?
It may be a good chance to re-think other charts as well? I can try to help there as needed :)

Reply:
Hi @LucaPrete !

Thank you very much for your help. We are currently discussing your proposal and thinking about the best way to tackle this, and make it a standard for the rest of charts as well. I will come back to you soon and provide with further info. 😄

Thanks for your feedback!

Reply:
Hi @LucaPrete,

We have discussed long about this considering your feedback and reached to the conclusion that we should keep our current approach of splitting into two values, `XXXX` and `externalXXXX`. Although it is true that merging into a single value would reduce the redundancy of information, our experience with previous cases tells us that it would be more difficult for the final to use. Having to separate entries, although redundant, seems to be clear and less confusing, so we should proceed this way in detriment of redundancy.

Nevertheless, we appreciate a lot your feedback and have discussed long this because we consider it extremely valuable.

Best regards!